tests/test_record_medal.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from router.record import verify_medal
+from models import VerifyBody
+
+
+class TestRecordMedal(unittest.TestCase):
+    def setUp(self):
+        self.default_verify = VerifyBody(sport_name="Archery", participants=[
+            {"country": "Thailand", "medal": {"gold": 1, "silver": 0, "bronze": 0}},
+            {"country": "China", "medal": {"gold": 0, "silver": 1, "bronze": 0}},
+            {"country": "Japan", "medal": {"gold": 0, "silver": 0, "bronze": 1}}, ])
+        self.repechage_verify = VerifyBody(sport_name="Boxing", participants=[
+            {"country": "Thailand", "medal": {"gold": 1, "silver": 0, "bronze": 0}},
+            {"country": "India", "medal": {"gold": 0, "silver": 1, "bronze": 0}},
```

Comment:
can you refactoring way to create medal pls, Hint: maybe create a method name `create_medal(gold: int, sliver: int, bronze: int) -> Dict[str]` and use it to create medal without nested uncomfortable  eye looking code like this, its good to create a method `create_medal_with_country` too